datatypes/src/main/java/org/hyperledger/besu/datatypes/Hash.java

```diff
@@ -29,6 +29,9 @@ public class Hash extends DelegatingBytes32 {
   /** The constant ZERO. */
   public static final Hash ZERO = new Hash(Bytes32.ZERO);
 
+  /** Last hash */
+  public static final Hash LAST = new Hash(Bytes32.fromHexString("F".repeat(64)));
```

Comment:
we already have that in the range Manager class RangeManager (MAX_RANGE and MIN_RANGE)

Reply:
I added here to DRY up a bit, and use it in RangeManager to define MAX_RANGE.